### PR TITLE
Revolution

### PR DESCRIPTION
**Pull request by EmyLightsaber**
Adding Revolution server to Servers.xml.

Pretty much a server for people from the various discords I am a member of, as well as the friends I've met along the way, family that are interested in playing, and The Internet in general should they wish to play.

### CODE REVIEW

**Comment by amoeba** [comment]
Hi @EmyLightsaber. Two questions:

1. Is the server up right now? I just tried and it looks down but I didn't check with an actual game client.
2. Do you have a Discord link you could add to your listing? If not, dropping some method of contact in this PR would be good, either your Discord profile, email address, etc.

**Comment by EmyLightsaber** [comment]
1. The server is indeed up right now, there's myself and another playing on it.
2. The Discord link is https://discord.gg/xaNB8pT8SC. How would I add it to the listing?

File: Servers.xml [hunk 1]
```diff
@@ -408,4 +408,16 @@
...
+    <id>f8fb8a21-82dc-4421-99d4-f93159d06f5d</id>
+    <name>Revolution</name>
+    <description>A server for Emys Friends, Family, Acquaintances, and The Internet</description>
+    <emu>ACE</emu>
+    <server_host>72.46.58.36</server_host>
+    <server_port>9000</server_port>
+    <type>PvE</type>
+    <status>Stable</stable>
+    <website_url></website_url>
+    <discord_url></discord_url>
```
Comment (amoeba) [inline Servers.xml:421]: @EmyLightsaber Add your Discord link (https://discord.gg/xaNB8pT8SC) in the `discord_url` element here.

**Review by amoeba — CHANGES_REQUESTED** [review]

**Comment by amoeba** [comment]
I added a note about where to put your Discord invite URL above. Once you do that I'll merge this.

**Comment by EmyLightsaber** [comment]
Did my request thing work? I'm very new at this.

**Comment by amoeba** [comment]
it sent a new PR which is fine. I've merged #126  and will just close this one. You're all set.

**Pull request closed without merging**